examples/solana/build_staking_operation.go

```diff
@@ -5,54 +5,37 @@ import (
 	"fmt"
 	"log"
 	"math/big"
-	"os"
-	"path/filepath"
-	"strings"
 	"time"
 
 	"github.com/btcsuite/btcutil/base58"
+	"github.com/coinbase/coinbase-sdk-go/gen/client"
 	"github.com/coinbase/coinbase-sdk-go/pkg/coinbase"
-
 	bin "github.com/gagliardetto/binary"
 	"github.com/gagliardetto/solana-go"
 	"github.com/gagliardetto/solana-go/rpc"
 )
 
 var (
-	networkID          = "solana-devnet"
-	amount             = big.NewFloat(0.1)
-	rpcURL             = "https://api.devnet.solana.com"
-	defaultPrivKeyPath = filepath.Join(home(), ".config/solana/id.json")
+	networkID = client.NETWORKIDENTIFIER_SOLANA_MAINNET
+	amount    = big.NewFloat(0.1)
+
+	apiKeyFilePath   = "YOUR_API_KEY"
+	rpcURL           = "RPC_URL"
+	walletAddress    = "YOUR_WALLET_ADDRESS"
+	walletPrivateKey = "YOUR_WALLET_PRIVATE_KEY"
 )
 
 func main() {
 	ctx := context.Background()
 
-	walletAddress := os.Args[2]
-
-	privKeys := []string{defaultPrivKeyPath}
-	if keys := os.Getenv("SOL_PRIVATE_KEYS"); keys != "" {
-		privKeys = strings.Split(keys, ",")
-	}
-
-	signers := make([]solana.PrivateKey, len(privKeys))
```

Comment:
This code wasn't functional when I tried using the env to pass `SOL_PRIVATE_KEYS` as below code attempts to read priv key from keygen file`PrivateKeyFromSolanaKeygenFile` instead of `PrivateKeyFromBase58` 

Have removed support for reading from file to keep things simple.

Reply:
I think this change makes sense 👍. No necessity to read this from a file.